File: day1/show_captured_video.py
import cv2

if __name__ == "__main__":
    id = 0  # camera id
    cap = cv2.VideoCapture(id)
    key = -1
    while key != 27:  # Esc
        ret, frame = cap.read()
        if ret:  # Display the resulting frame
            cv2.imshow('frame', frame)
        key = cv2.waitKey(25)
    cap.release()  # When everything done, release the video capture object
    cv2.destroyAllWindows()  # destroy all windows

File: day1/show_captured_video.cpp
#include <opencv2/opencv.hpp>
using namespace cv;
int main() {
	int id = 0;
	VideoCapture cap;
	cap.open(id, CAP_DSHOW);
	Mat frame;
	namedWindow("Original Video", WINDOW_AUTOSIZE);
	int key = -1;
	while (key != 27) {
		cap >> frame;
		imshow("Original Video", frame);
		key = waitKey(33);
	}
	return 0;
}
